src/models/transaction.cpp

```diff
@@ -3,7 +3,7 @@
 
 using namespace NickvisionMoney::Models;
 
-Transaction::Transaction(unsigned int id) : m_id{ id }, m_date{ boost::gregorian::day_clock::local_day() }, m_description{ "" }, m_type{ TransactionType::Income }, m_repeatInterval{ RepeatInterval::Never }, m_amount{ 0.00 }, m_groupId{ -1 }, m_rgba{ "#3584e4" }
+Transaction::Transaction(unsigned int id) : m_id{ id }, m_date{ boost::gregorian::day_clock::local_day() }, m_description{ "" }, m_type{ TransactionType::Income }, m_repeatInterval{ RepeatInterval::Never }, m_amount{ 0.00 }, m_groupId{ -1 }, m_rgba{ "" }
```

Comment:
I would still provide `m_rgba{ "rgb(53,132,228)" }` in case someone forgets to call `setRGBA`

Reply:
Then the default color will not be set in `transactiondialog.cpp`:
```
if(!gdk_rgba_parse(&color, m_controller.getRGBA().c_str()))
    {
        gdk_rgba_parse(&color, m_controller.getTransactionDefaultColor().c_str());
    }
```

Reply:
Oh yes you are right sorry.